Fixes 'spo eventreceiver remove' command

milanholemans opened the pull request:
Fixes 'spo eventreceiver remove' command. Closes #4033


### Remarks

Wrong endpoints were called. Reworked the entire command.

Adam-it reviewed (approved):
looks good. 
will do some local testing before I merge 👍

milanholemans commented:
@Adam-it one thing I forgot to update, I updated the scope of the telemetry, removed the anonymous value since it's a choice field. However, scope is an optional option so right now it's possible that `undefined` in the telemetry.
I'm unable to fix it myself right now since I'm on mobile, maybe you can do it before you merge it? 🙂

Adam-it commented:
> @Adam-it one thing I forgot to update, I updated the telemetry `scope` option, removed the anonymous value since it's a choice field. However, scope is an optional option so right now it's possible that `undefined` is logged in the telemetry. I'm unable to fix it myself right now since I'm on mobile, maybe you can do it before you merge it? 🙂

sure

Adam-it commented on src/m365/spo/commands/eventreceiver/eventreceiver-remove.ts:
```diff
@@ -52,10 +53,10 @@ class SpoEventreceiverRemoveCommand extends SpoCommand {
         listTitle: typeof args.options.listTitle !== 'undefined',
         listUrl: typeof args.options.listUrl !== 'undefined',
-        scope: typeof args.options.scope !== 'undefined',
+        scope: args.options.scope,
```
```suggestion
        scope: args.options.scope || 'site',
```
@milanholemans when not defined I will log the `site` scope as this seems to be the default one when this is left undefined.

milanholemans replied:
If I check the docs, `web` should be the default value.

> The scope. Can be either "site" or "web". Defaults to "web". Only applicable when not specifying any of the list properties.

Thank you for fixing this again ❤️
I totally forgot to fix it 😁

Adam-it replied:
sorry... web... totally got that the other way around looking at the old conditions before your change 😜.
no problem 👍

Adam-it commented:
added fix up
merged manually 
thanks @milanholemans for your awesome work 👍
your rock 🤩
